keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -220,7 +224,9 @@ know that this has succeeded?
 * Server should validate that no extra fields are present or invalid (e.g.
 misspelled), nor are any fields duplicated (for json and yaml data).
 * We must maintain compatibility with all existing clients, thus server side
-unknown field validation should be opt-in
+unknown field validation should be opt-in.
+* kubectl should provide the ability for a user to request server-side
+  validation.
```

Comment:
I expected kubectl to default to server-side validation against servers that support it

Reply:
Modified the wording of this bullet, let me know what you think.